src/test/viewport-spec.js

```diff
@@ -91,14 +90,13 @@ test('Viewport projection matrix', t => {
   t.end();
 });
 
-
-test('Viewport.project#2D', t => {
+test('Viewport.projectFlat', t => {
```

Comment:
Not really, there were some tricky lint warnings here so I had to rewrite.
We have added new projectFlat functions that ignore perspective, to match the old project ViewportMercatorProject functions so it seemed like the right thing to do.
Looks like it actually caused a regression though, this test case is failing now.